msg/SimulatorFeatures.msg

```diff
@@ -0,0 +1,30 @@
+# Features supported by the simulator.
+
+uint8 SPAWNING                  = 0      # Supports spawn interface (SpawnEntity).
+uint8 DELETING                  = 1      # Supports deleting entities (DeleteEntity).
+uint8 SPAWN_POSES               = 2      # Supports predefined spawn poses (GetSpawnPoses).
+uint8 SPAWN_BOUNDS              = 3      # Supports spawn bounds (SpawnableBounds).
+uint8 SPAWN_FORMAT_SDF          = 4      # Supports spawning from SDFormat specified in the uri field of SpawnEntity.
+uint8 SPAWN_FORMAT_URDF         = 5      # Supports spawning from Unified Robot Description Format.
+uint8 SPAWN_FORMAT_USD          = 6      # Supports spawning from Universal Scene Description format.
+uint8 SPAWN_FORMAT_NATIVE       = 7      # Supports spawning from prefab / other simulator native format.
```

Comment:
This here tells me it may be beneficial to block out a contiguous chunk of numbers for a potentially increasing number of formats. For example, MJCF :)